BetterShardsBukkit/src/main/java/vg/civcraft/mc/bettershards/database/DatabaseManager.java

```diff
@@ -221,6 +227,11 @@ public Boolean call() {
 					+ "inv_id INT NOT NULL,"
 					+ "last_upd TIMESTAMP NOT NULL DEFAULT NOW(),"
 					+ "PRIMARY KEY (uuid, inv_id));");
+		this.db.registerMigration(3, false, "CREATE TABLE IF NOT EXISTS portalVersion("
```

Comment:
This is redundant with the portal_type column in the table "createPortalDataTable". It also doesnt convert over any preexisting portals